[caffe2][easy] Save heap allocation in ConcatOp

swolchok opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #63529

Output() takes an IntArrayRef, so we can just use a std::initializer_list (stack-allocated array) instead of std::vector here.

Differential Revision: [D29687400](https://our.internmc.facebook.com/intern/diff/D29687400/)

swolchok committed:
[caffe2][easy] Save heap allocation in ConcatOp

Output() takes an IntArrayRef, so we can just use a std::initializer_list (stack-allocated array) instead of std::vector here.

Differential Revision: [D29687400](https://our.internmc.facebook.com/intern/diff/D29687400/)

[ghstack-poisoned]

swolchok committed:
Update on "[caffe2][easy] Save heap allocation in ConcatOp"


Output() takes an IntArrayRef, so we can just use a std::initializer_list (stack-allocated array) instead of std::vector here.

Differential Revision: [D29687400](https://our.internmc.facebook.com/intern/diff/D29687400/)

[ghstack-poisoned]

swolchok committed:
Update on "[caffe2][easy] Save heap allocation in ConcatOp"


Output() takes an IntArrayRef, so we can just use a std::initializer_list (stack-allocated array) instead of std::vector here.

Differential Revision: [D29687400](https://our.internmc.facebook.com/intern/diff/D29687400/)

[ghstack-poisoned]

swolchok commented:
ping -- need review

mruberry commented:
Do you think the segmentation fault in the ONNX build is related?

Aug 20 22:31:21 test/onnx/test_pytorch_onnx_onnxruntime.py::TestONNXRuntime_opset11::test_keypoint_rcnn Fatal Python error: Segmentation fault

mruberry reviewed (approved):
Neat

swolchok committed:
Update on "[caffe2][easy] Save heap allocation in ConcatOp"


Output() takes an IntArrayRef, so we can just use a std::initializer_list (stack-allocated array) instead of std::vector here.

Differential Revision: [D29687400](https://our.internmc.facebook.com/intern/diff/D29687400/)

[ghstack-poisoned]
